CMakeLists.txt

```diff
@@ -52,8 +59,32 @@ if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 endif()
 include(GNUInstallDirs)
 
+# If we aren't doing an app image build, configure runtime paths for relocatable installs
+if (VIAM_UR_DISABLE_APPIMAGE)
```

Comment:
This bit of boilerplate arranges things so that binaries in the prefix `bin` and `lib` directory will search for dependent libraries in what amounts to `../lib`. We need that for a non-appimage build so that the installation is relocatable, since the `module.tar.gz` will get unpacked into an unknown and varying location.

This may be temporary, depending on what our ultimate packaging solution looks like. If we end up with conan doing static links of dependencies, this can go away.